data/json/items/bionics.json

```
@@ -258,7 +258,7 @@
     "type": "BIONIC_ITEM",
     "name": { "str": "Cloaking System CBM" },
     "looks_like": "bio_int_enhancer",
-    "description": "A high-power system that uses a set of cameras and LEDs to make the user blend into their surroundings, rendering them fully invisible to eyes and optical sensors.  However, this doesn't prevent detection from other means such as infrared, sonar, etc.",
+    "description": "A high-power system that uses a set of cameras and LEDs to make the user blend into their surroundings, rendering them fully invisible to eyes and optical sensors.  However, this doesn't prevent detection from other means such as infrared, sonar, etc. Their manufacturer, Khan Defense Solutions, faced several lawsuits concerning mental illness stemming from extended use. However, these claims were litiated by the onset of rapture.",
```

```suggestion
    "description": "A high-power system that uses a set of cameras and LEDs to make the user blend into their surroundings, rendering them fully invisible to eyes and optical sensors.  However, this doesn't prevent detection from other means such as infrared, sonar, etc.  Their manufacturer, Khan Defense Solutions, faced several lawsuits concerning mental illness stemming from extended use.  However, these claims were overruled by the onset of rapture.",
```
I don't think you can use litigated like that, went with overruled as a still silly but more sensical alternative, using something not silly may be advisable instead tho